test/opf-var.jl

```diff
@@ -173,6 +173,29 @@ end
         end
     end
 
+    @testset "test ac opf" begin
+        @testset "5-bus uc storage case" begin
+            # work around possible bug in Juniper strong branching
+            result = PowerModels._run_uc_strg_opf("../test/data/matpower/case5_uc.m", ACPPowerModel, JuMP.with_optimizer(Juniper.Optimizer, nl_solver=JuMP.with_optimizer(Ipopt.Optimizer, tol=1e-4, print_level=0), branch_strategy=:MostInfeasible, log_levels=[]))
```

Comment:
Can the default version of `juniper_solver` work in this case?  I think this variant was a work around for an old bug that has since been fixed.

Reply:
Correct, it now works.

I also removed the extra juniper solver from https://github.com/lanl-ansi/PowerModels.jl/blob/cd543a09e2b8de5c2d2aee2059a64a03b9263b07/test/opf-var.jl#L157